SPECS/kata-packages/kata-packages.spec

```diff
@@ -0,0 +1,106 @@
+Summary:        Metapackage for Kata
+Name:           kata-packages-uvm
+Version:        1.1.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://aka.ms/mariner
+
+%description
+Metapackage for Kata.
+
+%package        uvm
```

Comment:
this is the main package, not kata-packages-uvm

Reply:
Updated.